src/chrome/content/rules/Vrt.be.xml

```diff
@@ -1,16 +1,10 @@
-<ruleset name="VRT">
+<ruleset name="VRT.be">
```

Comment:
Most of these already redirect to HTTPS, should there be rules for these as well? Or a comment?